grouper/models.py

```diff
@@ -514,7 +522,7 @@ def my_grantable_permissions(self):
                                    grantable[1] if len(grantable) > 1 else '*', ))
         return sorted(result, key=lambda x: x[0].name + x[1])
 
-    def my_groups(self):
+    def my_groups(self, ignore_enabled=False):
```

Comment:
Nit: I find `ignore_enabled` unintuitive, since I first thought that would filter out groups that were enabled.  Maybe `include_disabled` instead?